admin/webpack.config.js

```diff
@@ -80,7 +80,7 @@ module.exports = (env, argv) => {
       RETICULUM_SERVER: "hubs.local:4000",
       POSTGREST_SERVER: "",
       ITA_SERVER: "turkey",
-      TIER: "p0"
+      TIER: "p1"
```

Comment:
this is my own curiosity why do we need this one?

Reply:
Especially with alot of incoming work around theme customization, we use the admin panel theme tab a lot more now so it made sense to me to update this rather than manually do it each time.

Reply:
ah cool!